src/common/HTTPResponse.ts

```diff
@@ -68,13 +69,17 @@ export class HTTPResponse {
       ip: responsePayload.remoteIPAddress,
       port: responsePayload.remotePort,
     };
-    this._status = responsePayload.status;
+    // TODO extract statusText from extraInfo.headersText instead if present
```

Comment:
For my own understanding, is this a TODO you expect to wrap up before landing this patch, or a follow-up? (In case of the latter, let's file a tracking bug?)

Reply:
I copied it from Joey's PR https://github.com/puppeteer/puppeteer/pull/7640 so I plan to add him as a reviewer here.

Reply:
I think I tried to do this but ran into an issue. I'll try again and update with more context

Reply:
Ok, I just pushed a commit to my PR which takes care of this.